src/bantz/data/hybrid_retriever.py

```
@@ -0,0 +1,243 @@
+        """Expand seed nodes via graph traversal."""
+        expanded: Dict[str, RetrievalResult] = {}
+        seed_ids = {r.node.id for r in seeds}
+
+        for seed in seeds:
+            neighbours = await self._store.get_neighbors(
+                seed.node.id,
+                direction="both",
+                max_depth=self._max_depth,
+                min_weight=self._min_weight,
+            )
+            for i, nb in enumerate(neighbours):
```

Ruff (0.15.0)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)